### PR TITLE
chore: convert gradebook app to lab

### LINKED ISSUE #438: chore: convert gradebook workshop into lab project(JS only)
We currently have this review project on the beta JS curriculum
https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/#review-js-fundamentals-by-building-a-gradebook-app

Instead of having this in a step based form as a review project, it will work better as a lab
Go through the 4 steps and wip up some user stories we can use for this lab project
We can just keep the final solution for our tests. 

## Acceptance criteria

- [ ] create a new folder called gradebook app in the `frontend-cert/js-projects` directory
- [ ] add the completed project in an `script.js` file in that new folder
- [ ] create a markdown file for your proposed user stories. You should call it `user-stories.md` and place it in your new project folder

### PR DESCRIPTION
**Pull request by Dario-DC**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #438

### CODE REVIEW

File: frontend-cert/js-projects/gradebook-app/user-stories.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+1. You should have a function named `getAverage` that takes an array of test scores as a parameter and returns the average score.
+
+1. You should have a function named `getGrade` that takes a student score as a parameter and returns a string representing a letter grade based on the score. Here are the scores and their corresponding letter grades:
+
+    | Score Range | Grade   |
+    | ----------- | ------- |
+    | `100`       | `"A++"` |
```
Comment (moT01) [inline frontend-cert/js-projects/gradebook-app/user-stories.md:7]: I know this is how the existing project is, but is `A++` a thing? I think we should just go with `A+` - pretty minor. It's fine how it is.

File: frontend-cert/js-projects/gradebook-app/user-stories.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    | `90 - 99`   | `"A"`   |
+    | `80 - 89`   | `"B"`   |
+    | `70 - 79`   | `"C"`   |
+    | `60 - 69`   | `"D"`   |
+    | `0 - 59`    | `"F"`   |
+
+1. You should have a function named `hasPassingGrade` that takes a score as a parameter and returns either `true` or `false` depending on if the score corresponds to a passing grade. A passing grade is anything different from `"F"`. The function should use the `getGrade` function to determine the grade.
```
Comment (moT01) [inline frontend-cert/js-projects/gradebook-app/user-stories.md:14]:
```suggestion
1. You should have a function named `hasPassingGrade` that takes a score as a parameter and returns either `true` or `false` depending on if the score corresponds to a passing grade.

1. The `hasPassingGrade` function should use the `getGrade` function to get the letter grade, and use it determine if the grade is passing. A passing grade is anything different from `"F"`.
```

Pretty minor, but I would maybe split this into two user stories.

**Review by moT01 — COMMENTED** [review]
This is looking good @Dario-DC 👍 

I left a few suggestions. I don't think they need to be added, but I like it a little better. also, I know the issue says JS only, but I think it would be pretty easy to make a UI with this. A couple of inputs that call the functions when clicked. I think it just enhances the experience. I will leave a message in chat about that.

File: frontend-cert/js-projects/gradebook-app/user-stories.md (hunk 1 above)
Comment (Dario-DC) [reply frontend-cert/js-projects/gradebook-app/user-stories.md]: >is `A++` a thing?

I have no idea :smiley:

File: frontend-cert/js-projects/gradebook-app/user-stories.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    | `100`       | `"A+"`  |
+    | `90 - 99`   | `"A"`   |
+    | `80 - 89`   | `"B"`   |
+    | `70 - 79`   | `"C"`   |
+    | `60 - 69`   | `"D"`   |
+    | `0 - 59`    | `"F"`   |
+
+1. You should have a function named `hasPassingGrade` that takes a score as a parameter and returns either `true` or `false` depending on if the score corresponds to a passing grade.
+
+1. The `hasPassingGrade` function should use the `getGrade` function to get the letter grade, and use it determine if the grade is passing. A passing grade is anything different from `"F"`.
```
Comment (zairahira) [inline frontend-cert/js-projects/gradebook-app/user-stories.md:16]:
```suggestion
1. The `hasPassingGrade` function should use the `getGrade` function to get the letter grade, and use it to determine if the grade is passing. A passing grade is anything different from `"F"`.
```

**Review by zairahira — COMMENTED** [review]
The lab is looking good, no changes from my end 👍️

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Pull request merged**